libevmasm/Assembly.cpp

```diff
@@ -205,7 +205,8 @@ ostream& Assembly::streamAsm(ostream& _out, string const& _prefix, StringMap con
 	{
 		_out << _prefix << "stop" << endl;
 		for (auto const& i: m_data)
-			assertThrow(u256(i.first) < m_subs.size(), AssemblyException, "Data not yet implemented.");
```

Comment:
The first element should be a hash, so I guess it is practically impossible to do that. These problems will all go away when we transition to the new IR and bytecode generator.